Sources/Plugins/ExternalSensorsPlugin/BLE/DeviceHelper.swift

```diff
@@ -184,6 +184,7 @@ extension DeviceHelper {
     }
     
     func clearConnectedDevicesList() {
```

Comment:
It is not clear why this method needed and why it has public access? Looks like disconnectAllDevices should be enough and clear for callers.